.release-please-manifest.json

```diff
@@ -1,12 +1,12 @@
 {
```

Comment:
more PRs on the way for logging, might as well wait before merging this one

Reply:
Sounds like a plan. Are there any more coming or was the log bucket `disable: false` the last one?

Reply:
there's PR #686 in draft, then at least another one to regenerate README.md files

Reply:
Perfect, will hold off on hitting the merge button until those 2 are done.

Reply:
good to go now